pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -632,6 +634,23 @@ func emitSchedule(
 	return nil
 }
 
+// returns true if a schedule with the same label already exists
```

Comment:
nit: we usually start a comment with the methods name, so `checkScheduleAlreadyExists.returns true...`.

Another nit: period at the end of the comment.

Reply:
done